FindOnPage.md

```diff
@@ -0,0 +1,382 @@
+# WebView2Find API
+
+## Background
+
+The WebView2Find API provides methods and events to support text finding and navigation within a WebView2 control. 
+It allows developers to initiate find operations, navigate between find results, and customize various find configurations.
+
+## Examples
+
+
+#### Description
+To initiate a find operation within a WebView2 control, developers can utilize the `StartFindOnPage` method. 
+This method allows specifying the find term and configuring other find parameters using the `ICoreWebView2FindConfiguration` interface.
+
+#### Create/Specify a Find Configuration
+
+
+
+```cpp
+ICoreWebView2FindConfiguration CreateFindConfiguration()
```

Comment:
It's unclear to me whether this is the c++ or c# example - see the Scenario*.cpp files in WebView2APISample for example syntax on how to query interfaces and access properties for c++